tensorflow_datasets/core/download/download_manager.py

```diff
@@ -388,8 +388,14 @@ def manual_dir(self):
               self._manual_dir, self._manual_dir_instructions))
     return self._manual_dir
 
+  
+if sys.version_info[0] > 2:
```

Comment:
Now that this is always true, the condition can be removed.

Reply:
Yeah, I was also thinking of it . Thanks